camp/camp-brooklyn/src/main/java/org/apache/brooklyn/camp/brooklyn/BrooklynCampPlatform.java

```diff
@@ -61,6 +63,11 @@ public BrooklynCampPlatform(PlatformRootSummary root, ManagementContext manageme
         });
     }
 
+    /** finds and returns the {@link CampPlatform} registered for the management context, or null if none */
+    @Nullable public static CampPlatform findPlatform(ManagementContext mgmt) {
+        return mgmt.getConfig().getConfig(BrooklynCampConstants.CAMP_PLATFORM);
```

Comment:
Ah, I see you've changed `CampInternalUtils` to delegate to this! Given it's internal, then no strong feelings.